### PR TITLE
Mitra/DERC-2411/hide app gallery and app store if it is not supported 

### PR DESCRIPTION
**Pull request by mitra-deriv**
Changes:

- Hide app gallery and app store option in footer for deriv go for countries which is not supported

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/features/components/templates/footer/download.tsx [hunk 1]
```diff
@@ -9,27 +9,32 @@ import clsx from 'clsx'
...
-                    src="../../../../images/common/migration/footer/explore-appgallery.png"
-                    alt={localize('_t_explore it on appgallery_t_')}
-                    formats={['webp', 'auto']}
-                    width={90}
-                    placeholder="none"
-                />
-            </CustomLink>
+            {is_appgallery_supported && is_appgallery_and_ios_supported && (
+                <CustomLink
+                    href="https://appgallery.huawei.com/app/C103801913"
```
Comment (mayuran-deriv) [inline src/features/components/templates/footer/download.tsx:24]: dowe need to put this in const file?

File: src/features/components/templates/footer/download.tsx [hunk 2]
```diff
@@ -44,20 +49,22 @@ const DownloadBadges = () => {
...
-                    src="../../../../images/common/migration/footer/download-appstore.png"
-                    alt={localize('_t_download on the app store_t_')}
-                    formats={['webp', 'auto']}
-                    width={90}
-                    placeholder="none"
-                />
-            </CustomLink>
+            {is_ios_supported && is_appgallery_and_ios_supported && (
+                <CustomLink
+                    href="https://apps.apple.com/my/app/deriv-go-online-trading-app/id1550561298"
```
Comment (mayuran-deriv) [inline src/features/components/templates/footer/download.tsx:54]: const file?

File: src/features/components/templates/footer/download.tsx (hunk 1 above)
Comment (mitra-deriv) [reply src/features/components/templates/footer/download.tsx:24]: @mayuran-deriv  @Nuzhy-Deriv can confirm on if there was a specific reason they didn’t put it in the const ,as it should be same link we use for the banner in the deriv go page

Comment (Nuzhy-Deriv) [reply src/features/components/templates/footer/download.tsx:24]: no specific reason @mitra-deriv. If we are having these urls in a const file, we can use from there.

**Review by Nuzhy-Deriv — APPROVED** [review]

**Review by hasan-deriv — APPROVED** [review]

**Review by mayuran-deriv — APPROVED** [review]

**Review by meenakshi-deriv — APPROVED** [review]

**Pull request merged**